transforms/images/discard-border-objects-plugin/src/main.py

```diff
@@ -0,0 +1,113 @@
+import argparse, logging, os, time, filepattern
+from pathlib import Path
+import tensorflow as tf
+from functions import *
+
+
+def main(inpDir:Path, 
+         pattern:str,
+         groupBy:str,
+         outDir:Path,         
+         ):       
+        starttime= time.time()
+        if pattern is None:
+            logger.info(
+                    "No filepattern was provided so filepattern infer pattern from the input files"
+                )
+            try:
+                pattern = filepattern.infer_pattern([f.name for f in inpDir.iterdir()])
+            except ValueError:
+                logger.error(
+                    "Could not infer a filepattern from the input files, "
+                    + "and no filepattern was provided."
+                )
+                raise
```

Comment:
Since `filepattern` does not appear to be a necessity for this plugin, if no pattern is provided let's default to `.+`.